### PR TITLE
addict recipe

### PR DESCRIPTION
**Pull request by Ohjeah**

### CODE REVIEW

**Comment by jakirkham** [comment]
Please rebase. This stuff is already cleared.

File: recipes/addict/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{% set version = "1.0.0" %}
+{% set commit = "2ef6ab8e50df023984e76f9f3b404285afc4954d" %}
+
+package:
+  name: addict
+  version: {{ version }}
+
+source:
+    fn: {{ commit }}.tar.gz
+    url: https://github.com/mewwts/addict/archive/{{ commit }}.tar.gz
```
Comment (jakirkham) [inline recipes/addict/meta.yaml]: Could we please ask them in an issue to add a tag?

File: recipes/addict/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - addict 
+
+about:
+  home: https://github.com/mewwts/addict
+  license: MIT License
```
Comment (jakirkham) [inline recipes/addict/meta.yaml]: nit: please drop "License"

File: recipes/addict/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  summary: 'addict is a Python module that gives you a dictionary whose values are both gettable and settable using both attribute and getitem syntax.'
+
+extra:
+  recipe-maintainers:
+    - Ohjeah
+
```
Comment (jakirkham) [inline recipes/addict/meta.yaml]: nit: please strip extra newline.

File: recipes/addict/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    - python
+
+test:
+  imports:
+    - addict 
+
+about:
+  home: https://github.com/mewwts/addict
+  license: MIT License
+  summary: 'addict is a Python module that gives you a dictionary whose values are both gettable and settable using both attribute and getitem syntax.'
```
Comment (jakirkham) [inline recipes/addict/meta.yaml]: Could we please use this [short description]( https://github.com/mewwts/addict/blob/2ef6ab8e50df023984e76f9f3b404285afc4954d/setup.py#L8 )?

**Comment by Ohjeah** [comment]
Hi @jakirkham,

thanks for the remarks.

**Comment by jakirkham** [comment]
Of course.

If you wish to get it from PyPI instead, that would also be fine. Just if it comes from a repo, we like to have a tag.

**Comment by jakirkham** [comment]
LGTM

**Comment by jakirkham** [comment]
Thanks @Ohjeah.

**Comment by jakirkham** [comment]
@mewwts, just letting you know we have added `addict` to [conda-forge]( http://conda-forge.github.io/ ). This will allow users to easily install this with [`conda`]( http://conda.pydata.org/docs/ ) (`conda install -c conda-forge addict`). The maintenance burden here is light as all testing releases are built and deployed automatically with CI. Changes to how it is being built can be controlled by changing the recipe in [this repo]( https://github.com/conda-forge/addict-feedstock ). If you have any interest in being added as a maintainer (to make new releases for instance), we would love to have you on board. Thanks for your work on `addict`. 😄

**Pull request merged**